src/mumble/MainWindow.cpp

```diff
@@ -2073,6 +2074,16 @@ void MainWindow::sendChatbarText(QString qsText) {
 	qteChat->clear();
 }
 
+void MainWindow::sendChatbarPlainText(QString qsText) {
+	// Unify line endings
+	QString fixedNLPlain =
+		qsText.replace(QLatin1String("\r\n"), QLatin1String("\n")).replace(QLatin1String("\r"), QLatin1String("\n"));
```

Comment:
```suggestion
		QString fixedNLPlain = qsText.replace("\r\n", "\n").replace("\r", "\n");
```
For String literals we don't require any wrapper classes

Reply:
Saw several instances of this even in MainWindow.cpp and thought it was required. Removed in new commit.

Reply:
Yeah it used to be required but during the change to CMake we changed a few Qt settings which now no longer require this for string literals. We didn't update old code yet though.